README.fr.md

```diff
@@ -0,0 +1,145 @@
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_04.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_04.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_05.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_05.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png)
+[<img src="fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png" width=160>](fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png)
+[<img src="fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png" width=405>](fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png)
+[<img src="fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png" width=405>](fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png)
+
+## Description
+
+NewPipe n'utilise aucune librairie Google, ni l'API YouTube. 
+Les sites internets ne sont analysés que pour récupérer les informations nécessaires. Cette application peut donc être utilisée sans les services Google Play installés. De plus, vous n'avez pas besoin d'un compte YouTube pour utiliser NewPipe, qui est un logiciel libre copyleft.
+
+### Fonctionnalités
+
+* Rechercher des vidéos
+* Pas de compte nécessaire
+* Afficher les informations générales à propos de vidéos
+* Voir des vidéos YouTube
+* Écouter des vidéos YouTube
+* Mode flottant (lecteur flottant)
+* Choisir le lecteur avec lequel voir la vidéo
+* Télécharger des vidéos
+* Télécharger seulement le son des vidéos
+* Ouvrir une vidéo avec Kodi
+* Voir les vidéos suivantes/relatées
+* Rechercher sur YouTube dans un langage specifique 
+* Voir/Bloquer des contenus soumis à une restriction d'âge
+* Afficher des informations générales à propos des chaînes
+* Rechercher des chaînes
+* Regarder les vidéos d'une chaîne
+* Supporte Orbot/Tor (pas encore directement)
+* Supporte les résolutions 1080p/2K/4K
+* Voir l'historique
+* S'abonner à des chaînes
+* Rechercher dans l'historique
+* Rechercher/Voir des listes de lecture
+* Mettre et voir les listes de lecture en file d'attente
+* Mettre des vidéos en queue
+* Listes de lecture locales
+* Sous-titres
+* Voir des vidéos en direct
+* Afficher les commentaires
+
+### Supported Services
```

Comment:
Should this header be translated?

Reply:
Yeah, all headers should be translated. Don't forget to update the links at the top of the README.